archlinuxcn/kwin-decoration-sierra-breeze-enhanced/lilac.yaml

```
@@ -0,0 +1,8 @@
+maintainers:
+  - github: A1ca7raz
+    email: aya@wtm.moe
+
+update_on:
+  - source: github
+    github: kupiqu/SierraBreezeEnhanced
+    use_latest_release: true
```

`post_build` needed?

```suggestion
    use_latest_release: true

post_build: git_pkgbuild_commit
```